stetho/src/main/java/com/facebook/stetho/inspector/MethodDispatcher.java

```diff
@@ -63,6 +64,11 @@ public JSONObject dispatch(JsonRpcPeer peer, String methodName, @Nullable JSONOb
           "Not implemented: " + methodName,
           null /* data */));
     }
+
+    Stetho.getAnalyticsLogger().track(
+        "inspector_method_called",
```

Comment:
This I fear may expose more of an implementation detail in Stetho than actual feature usage.  For instance, Chrome is responsible for invoking these methods and it often does so eagerly.  For instance, on connect, Chrome "enables" every feature of the entire system with Stetho instead of waiting for the user to click around to various tabs and actually use them.  There are usually subtle clues after that that certain methods are responsible for usage but it could be hard to actually develop a heuristic to convert that to meaningful apples to apples usage numbers.

Likewise, this also suffers from not exposing the methods that Stetho invokes against Chrome (this tracks incoming method calls only).  However if you were to add that support it would highlight the problem even further as for example the vast majority of invokes Stetho does to Chrome would be around view tree diffing even if the tab isn't used at all.